specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/hyperdrive.json

```diff
@@ -0,0 +1,714 @@
+          "HyperDriveExperiment"
+        ],
+        "summary": "Create an Experiment.",
+        "description": "Create a HyperDrive Experiment.",
+        "operationId": "HyperDrive_CreateExperiment",
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ArmScopeParameter"
+          },
+          {
+            "in": "formData",
+            "name": "config",
+            "description": "The configuration file with experiment JSON content. A text file that is a JSON-serialized '#/definitions/HyperDriveCreateExperiment' object.",
+            "type": "file",
+            "required": true
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveExperimentResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a HyperDrive experiment": {
+            "$ref": "./examples/createHyperDriveExperiment.json"
+          }
+        }
+      }
+    },
+    "/hyperdrive/v1.0/{arm_scope}/runs/{run_id}/cancel": {
+      "post": {
+        "tags": [
+          "HyperDriveExperiment"
+        ],
+        "summary": "Cancel an Experiment.",
+        "description": "Cancel a HyperDrive Experiment.",
+        "operationId": "HyperDrive_CancelExperiment",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ArmScopeParameter"
+          },
+          {
+            "name": "run_id",
+            "in": "path",
+            "description": "Hyperdrive run id to cancel.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "header",
+            "name": "RunHistoryHost",
+            "description": "The host for run location.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveCancelExperimentResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/HyperDriveErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Cancel a HyperDrive experiment": {
+            "$ref": "./examples/cancelHyperDriveExperiment.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "HyperDriveErrorResponse": {
+      "properties": {
+        "code": {
+          "format": "int32",
```

Comment:
Is this really a number and not a string? Just double-checking (the Microsoft REST API Guidelines says to use a string)

Reply:
This is a number

Reply:
Changing to string will cause backward compat issues now.